src/test/scala/Covid19/JavaWrapperTest.scala

```diff
@@ -1,12 +1,29 @@
 package Covid19
 
+import Covid19.Protocol.CovidData
+
 class JavaWrapperTest extends org.scalatest.funsuite.AnyFunSuite {
+  val jw = new JavaWrapper()
+
+  test("return infected in the world") {
+    val worldFuture = jw.getInfectedInWorld.toCompletableFuture
+    while (!worldFuture.isDone) {}
+    val worldResult = worldFuture.get()
+    val russiaFromWorld = worldResult.items.find(_.isoCode.contains("RU"))
+
+    val russiaFuture = jw.getSummaryByCountry("RU").toCompletableFuture
+    while (!russiaFuture.isDone) {}
```

Comment:
wut? At least you can do some `unsafeRunSync` etc

Reply:
It's Java future. I have no other idea how to run it in sync way.